### DISCUSSION TITLE
M2O and M2M relational field display only relevant items only from other collection

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#11938] [Archive]
By configuring the right filter in the respective M2O or M2M field. 

display the related items of Collection 2 when we click collection1 item.

**Example**
![unknown (9)](https://user-images.githubusercontent.com/36895131/156613056-cbb20505-e8ab-4437-a78f-c5058c7c0202.png)

Tried but not working with current option. 

![unknown (10)](https://user-images.githubusercontent.com/36895131/156613196-fff132ad-277b-4cab-853e-3624c58f2ee8.png)
![unknown (11)](https://user-images.githubusercontent.com/36895131/156613201-61c970eb-fc91-44c8-9a31-2831b52cda0e.png)

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/11860
